.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,43 @@
+name: Feature request
+
+# See the json schema to help you create / update this form
+# https://json.schemastore.org/github-issue-forms.json
+
+description: Suggest an idea for this project
+
+title: "[ENH] "
+
+labels: ["feedback:feature request"]
```

Comment:
`feedback:feature request` is currently not an existing label. For now we can remove this, though we might want a longer discussion about issue labels in the future

Reply:
Modified to the new labels we created